src/sqlitestorage.cpp

```diff
@@ -1800,7 +1807,7 @@ bool SqliteStorage::loadNotebooks()
     sqlite3_stmt *stmt = NULL;
     bool isDefault;
 
-    Notebook::Ptr nb;
+    Notebook *nb;
```

Comment:
Minor detail, but I could move to the while() declaration to keep the scope minimal.

Reply:
As you suggested, I've turn the SqliteFormat method into returning a Notebook and not a pointer to one. I've updated here accordingly.